examples/create_rosa_cluster/create_rosa_sts_cluster/byo_oidc/cluster/README.md

```diff
@@ -0,0 +1,36 @@
+# ROSA STS cluster creation example
+
+This example shows how to create an STS _ROSA_ cluster. _ROSA_ stands for Red Hat Openshift Service on AWS
+and is a cluster that is created in the AWS cloud infrastructure.
+
+To run it:
+
+* Provide OCM Authentication Token 
+
+  OCM authentication token that you can get [here](https://console.redhat.com/openshift/token).
+    ```
+    export TF_VAR_token=...
+    ```
+
+* Provide OCM environment by setting a value to url    
+    ```
+    export TF_VAR_url=...
+    ```
+
+* Decide STS operator_role_prefix    
+    ```
+    export TF_VAR_operator_role_prefix=...
+    ```
+
+* Provide OIDC endpoint url
```

Comment:
Please describe how to provide it.... I mean how to create OIDC

Reply:
So that's the deal. That will be part of a separate PR, where we will add 'oidc_config' section in STS module